### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by BrightonMboya**
Initial Draft for the t3 docs. 
Kindly note: The home page is yet to be designed

### CODE REVIEW

File: docs/docs/Basics/Deployment.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+---
+sidebar_position: 5
+---
+
+#### Prerequisites
+
+1. We do not recommend deploying a SQLite database on serverless environments since the data wouldn't be persisted. I provisioned a quick Postgresql database on [Railway](https://railway.app), but you can of course use any other database provider. Make sure the prisma schema is updated to use the correct database.
```
Comment (juliusmarminge) [inline docs/docs/Basics/Deployment.md:7]: this sentence is kinda dumb now that tools like Turso exists. Should probably rephrase that it's the on-disk file-solution you shouldn't use in serverless 🤷🏼‍♂️

File: docs/docs/Basics/Installation.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+git clone https://github.com/t3-oss/create-t3-turbo.git
+```
+
+## Using the Turbo's CLI
+
+### npm
+
+```bash
+npx create-turbo@latest -e https://github.com/t3-oss/create-t3-turbo
+```
```
Comment (juliusmarminge) [inline docs/docs/Basics/Installation.md:19]: this doesn't work.

this repo requires PNPM

File: docs/docs/Basics/Quickstart.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+```bash
+npm install -g pnpm
+```
+
+or
+
+```bash
+npm install -g @pnpm/exe
+```
```
Comment (juliusmarminge) [inline docs/docs/Basics/Quickstart.md:21]: huh? never seen this before

Comment (juliusmarminge) [reply docs/docs/Basics/Quickstart.md:21]: fwiw I recommend using corepack to keep your pacakge manager version synced

File: docs/docs/Basics/Quickstart.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+```bash
+cp .env.example .env
+```
+
+For the env validation we use [T3-env](https://env.t3.gg/). When you add new environment variables, you should add the validation in the `packages/auth/env.mjs` file to make sure your app always runs with the correct env variables.
+
+## Databases
+
+Modify the `schema.prisma` file to use sqlite or any other provider of your choice
```
Comment (juliusmarminge) [inline docs/docs/Basics/Quickstart.md:41]: might be good with a note here saying that sqlite is generally for development only, or that you should use a hosted solution like turso if you want to keep sqlite in prod

File: docs/docs/Basics/folderStructure.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+### `.env.example`
+The `.env.example` file is used to provide an example of the environment variables that are required for the app to run. You should copy the file content to your own `.env` file and fill in the values. You can do it easily by running the following:
+
+```bash
+mv .env.example .env
+```
+
+### `.eslintrc.js`
+The `.eslintrc.cjs` file is used to configure ESLint. See [ESLint Docs↗](https://eslint.org/docs/latest/use/configure/configuration-files) for more information.
```
Comment (juliusmarminge) [inline docs/docs/Basics/folderStructure.md:81]: i removed this today as i did some restructuring of things.

might be good to look over this entire page to make sure it's valid

File: docs/docs/Usage/Expo Router.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+### `apps/expo/package.json`
+
+In here, you need to delete the entry point or replace it with `index.tsx`.
+
+```javascript
+{
+    "main": "index.tsx",
+    ...
+}
+```
```
Comment (juliusmarminge) [inline docs/docs/Usage/Expo Router.md:28]: ?

File: docs/docusaurus.config.js
Comment (juliusmarminge) [inline docs/docusaurus.config.js:1]: i'd prefer it if we use nextra but not a hard pass with docusaurus either.

also heard super good stuff about mintlify but never tried it and to be honest not 100% sure what it is xdd https://mintlify.com/docs/quickstart

**Comment by juliusmarminge** [comment]
Sorry for taking so long to get back to you on this. Just been super busy with other stuff that I haven't had time to look over this repo in a while...

File: docs/docs/Basics/Deployment.md (hunk 1 above)
Comment (BrightonMboya) [reply docs/docs/Basics/Deployment.md:7]: I copy pasted it from the ReadMe though, I guess we will have to remove from there as well. 
Will check the new state of the repo again and make the necessary changes

**Pull request closed without merging**